transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -35,6 +36,18 @@
     private final String name;
     private boolean removed;
 
+    // This will set to true once the ChannelHandler.handlerAdded(...) method was called. We need to keep track of this
```

Comment:
- Please use `/** ... */` because it's a field description.
- This will set to -> This is set to
- Please use `{@link ...}` where appropriate.